espresso-server/app/build.gradle.kts

```diff
@@ -59,6 +59,19 @@ android {
             }
         }
     }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
would that change still keep the compatibility with API 21, which we support as a minimum?

Reply:
I have run some e2e tests on a device with API 21 and they all pass. Is there any other check which can confirm compatibility?

Reply:
I would say that's enough. Thanks for checking